### PR TITLE
Settings parse and some typo fixes

### PR DESCRIPTION
**Pull request by slimus**
We should check and validate flag values after `flag.Parse()` cause now it always has default values.
@urbanishimwe please test it on your pc

- fix some typos in tests (`reponse` -> `request`)
- add more information to log string:
before: `PCAP Activate error: Cannot set as promisc`
after: `PCAP Activate device 'llw0' error: Cannot set as promisc`

### CODE REVIEW

File: settings.go [hunk 1]
```diff
@@ -242,6 +211,43 @@ func init() {
 	flag.Var(&Settings.modifierConfig.headerHashFilters, "output-http-header-hash-filter", "WARNING: `output-http-header-hash-filter` DEPRECATED, use `--http-header-hash-limiter` instead")
 
 	flag.Var(&Settings.modifierConfig.paramHashFilters, "http-param-limiter", "Takes a fraction of requests, consistently taking or rejecting a request based on the FNV32-1A hash of a specific GET param:\n\t gor --input-raw :8080 --output-http staging.com --http-param-limiter user_id:25%")
+
+	// default values, using for tests
+	Settings.outputFileConfig.sizeLimit = 33554432
+	Settings.outputFileConfig.outputFileMaxSize = 1099511627776
+	Settings.copyBufferSize = 5242880
+	Settings.inputRAWBufferSize = 0
```
Comment (urbanishimwe) [inline settings.go:219]: if you call `checkSettings` these value will be applied!
we can probably reduce lines of code?

Comment (slimus) [reply settings.go:219]: For tests it not applied. We can use default values like this or we should call checksettings in each test

Comment (urbanishimwe) [reply settings.go:219]: All may be we could call `checkSettings()` after `flagParse()`?

Comment (slimus) [reply settings.go:219]: In tests we don't have flags and don't call `flag.Parse()`

Comment (urbanishimwe) [reply settings.go:219]: Sounds Good now!

**Pull request merged**